frontend/app/src/components/display/amount/AmountDisplay.vue

```diff
@@ -368,6 +369,7 @@ const [DefineSymbol, ReuseSymbol] = createReusableTemplate<{ name: string }>();
         />
 
         <CopyTooltip
+          :disabled="privacyMode === PrivacyMode.PRIVATE"
```

Comment:
you can use `shouldShowAmount` from `const { shouldShowAmount } = storeToRefs(useSessionSettingsStore());`

equivalent to `privacyMode === PrivacyMode.PRIVATE`